### DISCUSSION TITLE
Stationary transformation?

### DISCUSSION THREAD

**Discussion by quannabe** [winedarksea/AutoTS#113] [Q&A]
Do I need to transform my time series so that it is stationary before passing to AutoTS?

**Comment by winedarksea** [comment]
No.
There's DifferencedTransformer (two of them stacked will make a second order differencing) and SeasonalDifference transformers, which are part of the preprocessing here and are the most common ways of making a series stationary. These will automatically be tried as part of the search process of the AutoTS class, and will be chosen if they help the model. 

Most models here don't need to be stationary, although it is often recommended for some of the Statsmodel models like DynamicFactor

**Reply by winedarksea** [reply]
Answering the question more generally, I would not recommend any preprocessing be applied before passing the series as input - let the model decide what is best.
The exception would be cleaning up known anomalies or outliers - while it will try automatic outlier removal, if you are aware of any you might be able to do a better job than the automatic outlier removal (ClipOutliers transformer is what I refer to here, which will be tried along with other preprocessing).

**Reply by quannabe** [reply]
Thank you!
